app/controllers/feedback_controller.rb

```diff
@@ -1,6 +1,6 @@
 class FeedbackController < ApplicationController
   def create
-    resource.assign_attributes params[:feedback]
+    resource.assign_attributes params.require(:feedback)
```

Comment:
Rails is getting stricter with regards to parameters from the controller, so we will need to use strong parameters like this in the future.  It also adds to the warm feeling of knowing what we are allowing through etc..